grc/core/Connection.py

```diff
@@ -79,6 +103,16 @@ def enabled(self):
         """
         return self.source_block.enabled and self.sink_block.enabled
 
+    @property
+    def label(self):
+        """ Returns a label for dialogs """
+        return 'Connection'
```

Comment:
idea: Mention the connection domain here

Reply:
![image](https://github.com/gnuradio/gnuradio/assets/508035/8cb57de1-f08c-4cb4-bc13-24b0685dd323)